robot/Cumulus/resources/NPSP.py

```diff
@@ -361,16 +361,21 @@ def fill_bge_form(self, **kwargs):
                 self.salesforce._populate_field(locator, value)
      
          
-    def verify_details_address(self,field,npsp_street, npsp_city, npsp_country):   
+    def verify_address_details(self,field,value,**kwargs):
         """Validates if the details page address field has specified value"""   
```

Comment:
please update the documentation to match the changed functionality of keyword

Reply:
done